### ISSUE TITLE
[RFE] Ship grafana dashboards used to represent indexed with `--reporting` mode

### ISSUE THREAD

**Issue by smalleni** [kube-burner/kube-burner#433]
**Is your feature request related to a problem? Please describe.**
It would be good to ship the Grafana dashboards we use for visualizing metric data shipped to ES.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by rsevilla87** [comment]
Agree, but, maybe we should consider using a different repo to avoid having an all-in-one repo?, not sure though, having the dashboards here would also bound them to an specific kube-burner version.

**Comment by vishnuchalla** [comment]
Now that we also opensearch integration in CI, I think we can setup a make target to ship grafana with some example dashboards using ES and prometheus as data sources.
